evalml/objectives/standard_metrics.py

```diff
@@ -63,7 +81,13 @@ def objective_function(self, y_true, y_predicted, X=None, sample_weight=None):
 
 
 class BalancedAccuracyMulticlass(MulticlassClassificationObjective):
-    """Balanced accuracy score for multiclass classification."""
+    """Balanced accuracy score for multiclass classification.
+
+    Example:
+        >>> y_true = pd.Series([0, 1, 0, 2, 0, 1, 2, 1, 2, 0, 2])
+        >>> y_pred = pd.Series([0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2])
+        >>> assert BalancedAccuracyMulticlass().objective_function(y_true, y_pred) == 0.5555555555555555
```

Comment:
Here I'm less worried about the significant figures because I think these objective definitions coming from sklearn are slow to change but our objective unit tests use `np.assert_almost_equal` so maybe we should too?